cwf/gateway/fabfile.py

```diff
@@ -48,7 +48,7 @@ def integ_test(gateway_host=None, test_host=None, trf_host=None,
                gateway_vm="cwag", gateway_ansible_file="cwag_dev.yml",
                transfer_images=False, destroy_vm=False, no_build=False,
                tests_to_run="all", skip_unit_tests=False, test_re=None,
-               run_tests=True):
+               count="1", run_tests=True):
```

Comment:
We have lots of different variables there. What about a more specific name like `iterations` or `num_iterations`. I know it is longer to type, but it may help to understand what kind of command we are running

Reply:
yeah iterations seems better. I just took `count` directly from the go test command. :)